Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9-slim
+
+ENV LECTURECUT_HOME=/LectureCut
+WORKDIR ${LECTURECUT_HOME}
+COPY . .
+
+RUN apt-get update && \
+    apt-get install -y gcc ffmpeg && \
+    pip install -r requirements.txt
```

Comment:
By setting `--no-cache-dir` we can decrease the size of the final image a bit.

Reply:
Done - good catch.